decentraland/unity-renderer
# feat: biw dev flag implement

## What does this PR change?

Fixes #1091

This PR has a refactor in the way builder in world works, it has been move all the HUDs to the BuilderInWorld plugin instead of handling it in the HUDController.

This way we can implement a dev Flag correctly that will work across all the project

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/biw-project-panel
2. Check that you are seeing a dev builder in world seeing the project section in the panel

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
